xroad-catalog-collector/packages/xroad-catalog-collector/redhat/SPECS/xroad-catalog-collector.spec

```diff
@@ -71,7 +71,7 @@ systemctl start postgresql
 
 #Check if database was already initialized
 if sudo -u postgres psql -lqt |cut -d \| -f 1 | grep -qw xroad_catalog ; then
-    echo "Database already initialized"
```

Comment:
Would be good to keep the echo but change the message for easier debugging

Reply:
I added a message